anti-analysis/packer/gopacker/packed-with-gopacker.yml

```
@@ -0,0 +1,28 @@
+rule:
+  meta:
+    name: packed with GoPacker
+    namespace: anti-analysis/packer/gopacker
+    author: jared.wilson@fireeye.com
+    description: The sample appears to be packed with GoPacker.
+    scope: file
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027.002]
+    mbc:
+      - Anti-Static Analysis::Software Packing::Standard Compression [OB0002.F0001.002]
+    references:
+      - https://github.com/petercunha/gopacker
+    examples:
+      - 4122acca2f9ea98fc3f3ad040688e4ce
+  features:
+    - and:
+      - string: "Anonymous"
+      - string: "<autogenerated>"
+      - string: "github.com/nirhaas/gopacker/"
+      - string: ".packed"
+        description: This is the default file extension given to packed files.
+      - string: "ZSTDCompression"
+        description: This packer is set to use ZSTD.
+      - string: "gopacker"
+      - string: "LALALALA"
+        description: LALALALA is a unique and default magic footer for this packing. This footer can be changed, however, by default it is not.
+      - count(string(LALALALA)): 2
```

```suggestion
      - optional:
        - string: "LALALALA"
          description: LALALALA is a unique and default magic footer for this packing. This footer can be changed, however, by default it is not.
```

if this is optional, let's mark it as such.